building/prerequisites.rst

```diff
@@ -16,5 +16,5 @@ target you will use in the end.
             expect flex ftp-upload gdisk iasl libattr1-dev libc6:i386 libcap-dev \
             libfdt-dev libftdi-dev libglib2.0-dev libhidapi-dev libncurses5-dev \
             libpixman-1-dev libssl-dev libstdc++6:i386 libtool libz1:i386 make \
-            mtools netcat python-crypto python3-pyelftools python-serial \
+            mtools netcat python-crypto python3-pyelftools python-serial python-wand \
```

Comment:
python-wand is not needed I guess? I think we used that when we had the TUI prototype in OP-TEE?

Reply:
Ok. It was in `aosp.rst` so included it here. Will remove it.